tests/utils.go

```diff
@@ -1336,6 +1367,14 @@ func SkipIfNoWindowsImage(virtClient kubecli.KubevirtClient) {
 	}
 }
 
+func SkipIfUseFlannel(virtClient kubecli.KubevirtClient) {
+	labelSelector := fmt.Sprint("app=flannel")
+	_, err := virtClient.CoreV1().Pods(metav1.NamespaceSystem).List(metav1.ListOptions{LabelSelector: labelSelector})
```

Comment:
I do not sure that it will returns err, when it does not have flannel pods, I think it will just return object with 0 items